system/jlib/jlog.ipp

```diff
@@ -633,6 +634,9 @@ protected:
     mutable CriticalSection   crit;
     mutable struct tm         startTime;
     bool printHeader = true;
+    long                      maxLogFileSize = 0;
+    long                      linesLoggedSinceSizeChecked = 0; // How lines logged since last size check for rollover
+    long                      sizeCheckInterval = 0;
```

Comment:
picky: use of long variables is a bit unusual, and sizeCheckInterval isn't very descriptive.